src/main/controllers/PendingSubscriptionsController.ts

```diff
@@ -7,9 +7,10 @@ const subscriptionService = new SubscriptionService();
 
 export default class PendingSubscriptionsController {
   public async get(req: PipRequest, res: Response): Promise<void> {
+    console.log('get', req.user['oid']);
```

Comment:
left console log in

Reply:
this could be removed

Reply:
I noticed it as well and removed it along enabling e2e on staging